package.json

```diff
@@ -78,12 +78,13 @@
     },
     "showProviderName": {
       "type": "boolean",
-      "description": "Show provider name in messages",
+      "description": "Show provider name in bubble. It's always shown in the panel",
```

Comment:
The name should be consistent here, other options refer to this as the "Tooltip", but here it is called "bubble".